unicodetools/pom.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>unicodetools</artifactId>
+	<groupId>org.unicode.unicodetools</groupId>
+	<name>Unicode Tools</name>
+
+	<version>0.0.1</version>
+
+	<parent>
+		<groupId>org.unicode.unicodetools</groupId>
+		<artifactId>unicodetools-parent</artifactId>
+		<version>0.0.0</version>
+	</parent>
+
+
+	<url>https://unicode.org/cldr</url>
+
+	<properties>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<!-- <mainClass>org.unicode.cldr.tool.Main</mainClass> -->
```

Comment:
Here and below: Is it safe to remove the commented-out portions in this file?

Reply:
yes, will do